src/SMG/ManagerBundle/Controller/Traits/HandleUserTrait.php

```diff
@@ -34,4 +37,28 @@ private function validates($user, $group)
 
         return $errors;
     }
+
+    /**
+     * Check if the JSON sent data is correct for
+     * the current called action and throws a
+     * bad request exception if the input is wrong.
+     *
+     * @return array
+     */
+    private function requestIsJsonWithKeysOrThrow(
```

Comment:
should be in an other trait, because it has nothing to do with specificall handling user